### PR TITLE
add-copyright to the bottom of the page

### PR DESCRIPTION
**Pull request by Ariacoding**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -37,3 +37,10 @@ body {
   right: 0px;
   top: 0px;
 }
+.Copyright {
```
Comment (iheidari) [inline index.css]: Make it all lowercase

Comment (Ariacoding) [reply index.css]: > Make it all lowercase

sure

**Review by iheidari — APPROVED** [review]
Good job

**Comment by Ariacoding** [comment]
> Good job

thx

**Pull request merged**